Automattic/jetpack
# Backup: Add tracking when user clicks on upgrade storage prompts

The idea is to track when a user clicks on our new upgrade storage prompts introduced in #28581.

## Proposed changes:

* Add new Track event when user click on upsell prompts CTA.

## Screenshots:
| Component | Event | Screenshot |
|---|---|---|
| Storage upgrade prompt CTA. It appears when the used storage is 65% or above. | `jetpack_backup_upgrade_storage_prompt_cta` | ![image](https://user-images.githubusercontent.com/1488641/215570072-206ea811-602e-4e35-b116-46bfd13382f4.png) |
| Upgrade your storage button on header. It appears when the used storage is 100% or above. | `jetpack_backup_upgrade_storage_header_cta` | ![image](https://user-images.githubusercontent.com/1488641/215570393-5419f3f3-586d-4092-a7dd-0f9989d3e032.png) |

### Other information:


## Jetpack product discussion


peaFOp-wy-p2

## Does this pull request change what data or activity we track or use?


Yes. It tracks when a user clicks on `Upgrade storage` prompts.

## Testing instructions:






* Using a site with storage full, navigate to Jetpack -> VaultPress Backup
* Open developer console in Network tab and ensure `Preserve log` are ticked.
* Click on the upgrade prompt that appears below the storage meter. It should redirect you to the checkout page
* You should see a `t.gif` call with the event `jetpack_backup_upgrade_storage_prompt_cta`.
* Repeat the process but clicking the `Upgrade your storage` button at top.
* You should see a `t.gif` call with the event `jetpack_backup_upgrade_storage_header_cta`.
